tests/test_listener.py

```diff
@@ -50,7 +50,7 @@ def test_dynamic_attributes(self, mock_client_init, mock_listener,
     @mock.patch(REPORT_PORTAL_SERVICE)
     def test_non_critical_test_skip(self, mock_client_init, mock_listener,
                                     test_attributes):
-        test_attributes['critical'] = 'no'
+        test_attributes['critical'] = False
```

Comment:
Unfortunately you can't fix the unit-test like that, Robot Framework sets this value to `yes`/`no`